src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ContentDbContext.cs

```diff
@@ -86,13 +87,16 @@ private void Configure(bool updateTimestamps = true)
         public virtual DbSet<UserReleaseInvite> UserReleaseInvites { get; set; }
         public virtual DbSet<UserPublicationInvite> UserPublicationInvites { get; set; }
 
-        public virtual IQueryable<FreeTextRank> PublicationsFreeTextTable(string search) =>
-            FromExpression(() => PublicationsFreeTextTable(search));
+        [DbFunction]
+        public virtual IQueryable<FreeTextRank> PublicationsFreeTextTable(string searchTerm) =>
+            FromExpression(() => PublicationsFreeTextTable(searchTerm));
+
+        [DbFunction]
+        public virtual IQueryable<FreeTextRank> ReleaseFilesFreeTextTable(string searchTerm) =>
+            FromExpression(() => ReleaseFilesFreeTextTable(searchTerm));
 
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
```

Comment:
I've checked it now and it's a good job you raised this because removing that line did cause a problem for new migrations.

I have reinstated it in d82b025bd2b58acb3b053a2452d6ad268ea9e2f3 using `IEntityTypeConfiguration` format.